.travis.yml

```diff
@@ -26,17 +26,17 @@ env:
 
 matrix:
   include:
-    - php: '7.1'
+    - php: 7.1
       env: DB=MYSQL RECIPE_VERSION=4.5.x-dev PHPUNIT_TEST=1 PHPCS_TEST=1
-    - php: '7.1'
-      env: DB=PGSQL RECIPE_VERSION=4.5.x-dev PHPUNIT_COVERAGE_TEST=1
-    - php: '7.2'
-      env: DB=MYSQL RECIPE_VERSION=4.5.x-dev PHPUNIT_TEST=1
-    - php: '7.3'
+    - php: 7.1
+      env: DB=MYSQL RECIPE_VERSION=4.5.x-dev PHPUNIT_COVERAGE_TEST=1 PDO=1
+    - php: 7.2
+      env: DB=PGSQL RECIPE_VERSION=4.6.x-dev PHPUNIT_TEST=1
+    - php: 7.3
       env: DB=MYSQL RECIPE_VERSION=4.6.x-dev BEHAT_TEST=1
-    - php: '7.3'
+    - php: 7.3
       env: DB=MYSQL RECIPE_VERSION=4.6.x-dev NPM_TEST=1
-    - php: '7.3'
```

Comment:
FYI, the quotes are deliberate, as Travis's config schema defines these as strings, and their linting tool complains about it